roles/lib_utils/build/generate.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+'''
+  Generate the openshift-ansible/roles/lib_openshift_cli/library/ modules.
+'''
+
+import os
+
+# pylint: disable=anomalous-backslash-in-string
+GEN_STR = "#!/usr/bin/env python\n"                                  + \
+          "# pylint: disable=missing-docstring\n"                    + \
+          "#     ___ ___ _  _ ___ ___    _ _____ ___ ___\n"          + \
+          "#    / __| __| \| | __| _ \  /_\_   _| __|   \\\n"        + \
+          "#   | (_ | _|| .` | _||   / / _ \| | | _|| |) |\n"        + \
+          "#    \___|___|_|\_|___|_|_\/_/_\_\_|_|___|___/_ _____\n"  + \
+          "#   |   \ / _ \  | \| |/ _ \_   _| | __|   \_ _|_   _|\n" + \
+          "#   | |) | (_) | | .` | (_) || |   | _|| |) | |  | |\n"   + \
+          "#   |___/ \___/  |_|\_|\___/ |_|   |___|___/___| |_|\n"
+
+OPENSHIFT_ANSIBLE_PATH = os.path.dirname(os.path.realpath(__file__))
+
+FILES = {'yedit.py':  ['doc/license', 'doc/yedit', 'src/base.py', 'src/yedit.py', 'ansible/yedit.py'],
```

Comment:
Also, updated the gist w/ an example of using extra licenses, etc.
https://gist.github.com/tbielawa/abc2dde72cc51eb2f1c6ff01b6c780b0#file-generate-special-license-yaml